### PR TITLE
Feature/get block locs

### PR DESCRIPTION
**Pull request by jake-nyquist**
Added a new PR for this because the old had a bunch of code in it.

### CODE REVIEW

File: zookeeper/source/zk_nn_client.cc [hunk 1]
```diff
@@ -1210,26 +1223,46 @@ void ZkNnClient::get_block_locations(const std::string &src,
...
-          LOG(INFO) << "Block DN Loc: " << *data_node;
-          buildDatanodeInfoProto(located_block->add_locs(), *data_node);
+        // Add block indices for an EC block.
+        // Each byte (i.e. char) represents an index into the group.
+        std::string block_index_string;
+        int i = 0;
+        // Add datanodes for the EC block
+        for (auto data_node : data_nodes) {
+          auto data = std::vector<uint8_t>();
+          if(!zk->get(BLOCK_GROUP_LOCATIONS + data_node, data,
```
Comment (jake-nyquist) [inline zookeeper/source/zk_nn_client.cc]: Went ahead and read the datanode ip off of the znode here @adamnsm1 @riceluxs1t

**Comment by jake-nyquist** [comment]
@riceluxs1t @adamnsm1 can you review so I can merge

**Review by adamnsm1 — APPROVED** [review]
I'm not that familiar with all the proto stuff, but assuming that's right everything looks good to me

**Comment by ghost** [comment]
deleting this PR because the functionality was reimplemented in feature/ectests3

**Pull request closed without merging**